doc/admin-guide/files/records.config.en.rst

```diff
@@ -3477,6 +3477,14 @@ URL Remap Rules
    Set this variable to ``1`` if you want to retain the client host
    header in a request during remapping.
 
+.. ts::cv CONFIG proxy.config.url_remap.min_rules_required INT 1
```

Comment:
`ts::cv` -> `ts:cv`  (one colon)

Reply:
Not quite - should be `ts:cv::`

Reply:
I think this will address the doc build rule:
https://ci.trafficserver.apache.org/job/Github_Builds/job/docs/905/console